core/sdk/src/api/manifest/device_manifest.rs

```diff
@@ -372,6 +377,7 @@ impl Default for DefaultValues {
             allow_watch_history: false,
             skip_restriction: "none".to_string(),
             video_dimensions: default_video_dimensions(),
+            accessibility_audio_description_settings: false,
```

Comment:
Use default_accessibility_audio_description_settings()  here, otherwise remove this function.